index.html

```diff
@@ -10,38 +10,55 @@
 </head>
 
 <body>
-
+    <section class="main-container">
     <header class="main-header">
-        <nav class="nav-bar">
+        <nav class="nav-bar" >
             <div class="nav-img">
                 <img src="images/image_geometry_1.png" alt="geometry -img">
             </div>
             <div class="hamburger">
-                <img src="images/ic_menu.jpg" alt="hamburger">
+                <span class="line"></span>
+                <span class="line"></span>
+                <span class="line"></span>
+            </div>
+            <div class="hamburger-menu">
+                <ul>
+                    <li><a href="#">Hello</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
             </div>
 
         </nav>
     </header>
     <section class="homepage">
-        
+                 
         <div class="homepage-container">
             <h1 class="homepage-heading">Emmanuel <br>
                 Tofa</h1>
```

Comment:
- [Optional] Consider using an indentation of two spaces in all your documents. This is a conventional and also best practice.